subsys/nrf_security/src/drivers/cracen/silexpk/target/baremetal_ba414e_with_ik/pk_baremetal.c

```diff
@@ -195,9 +197,10 @@ struct sx_pk_acq_req sx_pk_acquire_req(const struct sx_pk_cmd_def *cmd)
 	req.req->cmd = cmd;
 	req.req->cnx = &silex_pk_engine;
 
+#ifdef CONFIG_CRACEN_HW_VERSION_BASE
 	cracen_acquire();
```

Comment:
We have a global CRACEN mutex which counts the users and enables disables cracen. Will that affect it?

If it does you can avoid ifdefing this and ifdef the code which accesses the interrupt in the cracen.c?

Reply:
Discussed offline. `cracen_acquire` should not be `#ifdef` out. That solves the problem